dev/com.ibm.ws.jaxrs.2.1.client_fat/fat/src/com/ibm/ws/jaxrs21/client/fat/test/JAXRS21ClientSSLProxyAuthTest.java

```diff
@@ -398,7 +375,7 @@ public void testProxyReturnsAuthError_WebTarget() throws Exception {
         assertEquals("Proxy Password value printed in trace output", 0, server.findStringsInLogsAndTraceUsingMark("myPa\\$\\$word").size());
     }
 
-    // @Test // TODO: Enable after fixing Open-Liberty issue 169
+//    @Test // TODO: Enable after fixing Open-Liberty issue 169
```

Comment:
Looks like issue #169 is fixed.  Can you try re-enabling this test to see if it passes now?

Reply:
I looked into that but wanted to keep it separate from this pull request, particularly since I'm having trouble running the entire bucket locally.